src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const totalCost = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return totalCost - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return totalCost - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
The logic for applying discounts is incorrect. If the number of days is greater than or equal to the long-term threshold, the function should apply the long-term discount and not check for the short-term discount. However, if the number of days is less than the long-term threshold but greater than or equal to the short-term threshold, then the short-term discount should be applied. Consider restructuring the conditional statements to ensure only the appropriate discount is applied.

Reply:
Please clarify